packages/medusa/src/api/routes/admin/product-categories/delete-products-batch.ts

```diff
@@ -0,0 +1,121 @@
+import { IsArray, ValidateNested } from "class-validator"
+import { Request, Response } from "express"
+
+import { EntityManager } from "typeorm"
+import { ProductBatchProductCategory } from "../../../../types/product-category"
+import { ProductCategoryService } from "../../../../services"
+import { Type } from "class-transformer"
+import { FindParams } from "../../../../types/common"
+
+/**
+ * @oas [delete] /product-categories/{id}/products/batch
+ * operationId: "DeleteProductCategoriesCategoryProductsBatch"
+ * summary: "Delete Products"
+ * description: "Remove a list of products from a product category."
+ * x-authenticated: true
+ * parameters:
+ *   - (path) id=* {string} The ID of the Product Category.
+ *   - (query) expand {string} (Comma separated) Category fields to be expanded in the response.
+ *   - (query) fields {string} (Comma separated) Category fields to be retrieved in the response.
+ * requestBody:
+ *   content:
+ *     application/json:
+ *       schema:
+ *         $ref: "#/components/schemas/AdminDeleteProductCategoriesCategoryProductsBatchReq"
+ * x-codegen:
+ *   method: removeProducts
+ * x-codeSamples:
+ *   - lang: Shell
+ *     label: cURL
+ *     source: |
+ *       curl --location --request DELETE 'https://medusa-url.com/admin/product-categories/{id}/products/batch' \
+ *       --header 'Authorization: Bearer {api_token}' \
+ *       --header 'Content-Type: application/json' \
+ *       --data-raw '{
+ *           "product_ids": [
+ *             {
+ *               "id": "{product_id}"
+ *             }
+ *           ]
+ *       }'
+ * security:
+ *   - api_token: []
+ *   - cookie_auth: []
+ * tags:
+ *   - Product Category
+ * responses:
+ *   200:
+ *     description: OK
+ *     content:
+ *       application/json:
+ *         schema:
+ *           type: object
+ *           properties:
+ *             product_category:
+ *               $ref: "#/components/schemas/ProductCategory"
```

Comment:
todo: We should create a unified return type in oas for this response

Reply:
Made the change here - https://github.com/medusajs/medusa/pull/3141/commits/814d4d06f054e2b1cf796d44d947c5f8924037b7, comment here if that wasn't the intention. I can open a new PR.